config/csm-common.mk

```diff
@@ -27,7 +27,7 @@ CSM_BASEIMAGE=quay.io/dell/container-storage-modules/csm-base-image:nightly
 DEFAULT_BASEIMAGE=${UBI_BASEIMAGE}
 
 # --- DEFAULT_GOVERSION: Specifies the default version of go.
-DEFAULT_GOVERSION=1.23
+DEFAULT_GOVERSION="1.23"
```

Comment:
This is a Makefile variable and so should not be quoted.

Reply:
Fixed in https://github.com/dell/common-github-actions/pull/130